src/test/java/uk/ac/cam/cl/dtg/isaac/api/ITConstants.java

```diff
@@ -70,6 +92,38 @@ public final class ITConstants {
   public static final String QUIZ_TEST_QUIZ_ID = "_quiz_test";
   public static final String QUIZ_HIDDEN_FROM_ROLE_STUDENTS_QUIZ_ID = "_hidden_from_roles_student_quiz_test";
   public static final String QUIZ_HIDDEN_FROM_ROLE_TUTORS_QUIZ_ID = "_hidden_from_roles_tutor_quiz_test";
+  public static final String UNKNOWN_QUIZ_ID = "_not_a_quiz";
+
+  // Assignments
+  public static final Long ASSIGNMENT_FACADE_TEST_GROUP_ID = 5L;
+
+  public static final Long QUIZ_ASSIGNMENT_ID = 1L;
+  public static final Long QUIZ_ASSIGNMENT_FOR_CANCELLATION_TEST_FIRST_ID = 2L;
+  public static final Long QUIZ_ASSIGNMENT_FOR_CANCELLATION_TEST_SECOND_ID = 3L;
+  public static final Long QUIZ_ASSIGNMENT_SECOND_ID = 4L;
+  public static final Long QUIZ_ASSIGNMENT_CANCELLED_ID = 5L;
+  public static final Long QUIZ_ASSIGNMENT_EXPIRED_ID = 6L;
+  public static final Long QUIZ_ASSIGNMENT_FEEDBACK_MODE_ID = 7L;
+  public static final Long QUIZ_ASSIGNMENT_SET_INCOMPLETE_TEST_ID = 8L;
+  public static final long QUIZ_ASSIGNMENT_NON_EXISTENT_ID = 1000L;
```

Comment:
Another really small one, but shouldn't this be `Long`, just for consistency? There are also three spaces after this. Does that mean that a comment is missing there?